service-commands/src/commands/users.js

```diff
@@ -11,7 +11,21 @@ User.addUser = async (libsWrapper, metadata) => {
   return userId
 }
 
+// TODO: we shouldn't need to upload the photo prior to signup as this is not
+// representative of our actual flow. But since the third party libs gets confused
+// uploading a photo via code, have to do this work around.
 User.uploadProfileImagesAndAddUser = async (libsWrapper, metadata, userPicturePath) => {
+  // Sign user up
+  const userId = await User.addUser(libsWrapper, metadata)
+
+  // Wait for disc prov to index user
+  await waitForIndexing()
+
+  // Get primary off user and set in libs
+  const user = await User.getUser(libsWrapper, userId)
```

Comment:
actually may not be necessary bc signup should cover setting the cn endpoint to the new primary